app/views/backoffice/auth0/index.en.html.erb

```diff
@@ -1,11 +1,11 @@
 <% title 'Back office login' %>
 
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <h1 class="heading-xlarge gv-u-heading-xxlarge">Back office</h1>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Back office</h1>
 
-    <p class="lede">You need to authenticate in order to access the back office.</p>
+    <p class="govuk-body-l">You need to authenticate in order to access the back office.</p>
 
-    <%= button_to 'Login', admin_auth_url, method: :post, class: 'button', id: 'qsLoginBtn' %>
+    <%= button_to 'Login', admin_auth_url, method: :post, class: 'button' %>
```

Comment:
Can we also migrate the button?
Example here: app/views/layouts/_developer_tools.html.erb